src/olympia/blocklist/tests/test_serializers.py

```diff
@@ -8,6 +12,7 @@
 
 class TestBlockSerializer(TestCase):
```

Comment:
It's not clear to me from the diff but make sure there is a test covering the scenario where all are blocked and addon is NOT disabled.

Reply:
The test for that scenario was missing in the existing tests, looks like.  I'll add it for completeness.